### PR TITLE
Update Pull Request Template to be More Helpful

### PR DESCRIPTION
**Pull request by zoracon**
Would like feed back on whether or not this is more helpful than the current template!

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,26 +1,19 @@
...
+## Type of change
 
-- pr link
+Please delete options that are not relevant.
 
-### List related issues if any
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Refactoring existing code
+- [ ] New Ruleset (fix or feature that would cause existing functionality to not work as expected)
```
Comment (Bisaloo) [inline .github/pull_request_template.md:10]: I don't understand the purpose of the precision in parentheses here. Why couldn't a new ruleset just aim at increasing coverage? Maybe it's just the phrasing.

Comment (zoracon) [reply .github/pull_request_template.md:10]: Good catch, didn't realize I left the "default template" language that I borrowed from in here. I will remove that.

Comment (zoracon) [inline .github/pull_request_template.md:10]:
```suggestion
- [ ] New Ruleset
```

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -1,26 +1,19 @@
...
+- [ ] Refactoring existing code
+- [ ] New Ruleset
+- [ ] Existing Ruleset
 
-- issue link
+## For Rulesets: Testing
 
-***May delete note after reading***
+- [ ] Ran `./test.sh` to test coverage for ruleset
+- [ ] Ran `python3.6 test/rules/src/https_everywhere_checker/check_rules.py  test/rules/http.checker.config src/chrome/content/rules/<new-ruleset.xml>` to check for errors in domains
```
[the remark this answers is not in the document]
Comment (zoracon) [reply .github/pull_request_template.md:16]: Good point! Thanks for catching my GNU/Linux bias here. Do you feel we should have a comprehensive list here, link to documentation for testing, or just leave it out?

[the remark this answers is not in the document]
Comment (J0WI) [reply .github/pull_request_template.md:16]: Or maybe just make sure that the CI completes without errors?

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -1,26 +1,19 @@
-***Delete section if irrelevant***
+Fixes #(issue)
 
-### List related PRs if any
+## Type of change
 
-- pr link
+Please delete options that are not relevant.
```
Comment (cschanaj) [inline .github/pull_request_template.md:5]: Given the type of change is a collection of checkbox, isn't it more natural to expect the contributor to "toggle the relevant options/ checkbox"?

File: .github/pull_request_template.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (zoracon) [reply .github/pull_request_template.md:16]: @J0WI That works. Sometimes people seem to be confused about the CI, so I'll just put advice for that. I have a script locally where I parse the fetch errors, maybe I can put that up if that is needed later.

File: .github/pull_request_template.md (hunk 3 above)
Comment (zoracon) [reply .github/pull_request_template.md:5]: That is more concise, I will remove this line

**Review by ivysrono — APPROVED** [review]

**Pull request merged**